### PR TITLE
DO-7433 : fix to not able to upload SECOND_SERVICE docker image on nexus.

### PR DESCRIPTION
**Pull request by ankushnisal**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,6 +22,12 @@ clean:
 docker:
 	@echo ">> Compile using docker container"
 	@docker build -t "$(DOCKER_IMAGE_NAME)" .
+	@echo $NEXUS_PASSWORD | docker login -u $NEXUS_USER --password-stdin $NEXUS_SERVER
+	@echo $AWS_PASSWORD | docker login -u AWS --password-stdin AWS_ECR_REGISTRY
```
Comment (namkyvo) [inline Makefile:26]: Change AWS_PASSWORD to AWS_ECR_TOKEN

Comment (ankushnisal) [reply Makefile]: Updated as per review comment

**Review by namkyvo — COMMENTED** [review]
can you check if TAG is not empty ?

**Review by hgokhale — APPROVED** [review]

**Review by hgokhale — CHANGES_REQUESTED** [review]
Actually we should not add the code to push image over here. This repo os not built through jenkins.. this is an external repo and at some point we may want to merge this back to the external repo that we forked from.

We should push the image from the translator-nutanix repo build

**Pull request closed without merging**